src/Header/Header.js

```diff
@@ -676,6 +676,8 @@ module.exports = kind(
 					: (this.get('title') || this.get('content')),
 			subtitle = this.get('titleBelow');
 		if ((this.get('type') == 'small') && subtitle) {
+			title = this.allowHtml? title: dom.escape(title);
+			subtitle = this.allowHtml? subtitle: dom.escape(subtitle);
```

Comment:
Is there a reason for using a ternary operation on these two lines instead of just a regular 'if' statement? If this.allowHtml is true then they set title = title and subtitle = subtitle. Why not `if (!this.allowHtml) {title =  dom.escape(title)}` ?

Also, as an aside, the ternary operations should probably have a space before the question marks and colons.